src/components/NotificationsDrawer/DrawerPanel.tsx

```diff
@@ -0,0 +1,515 @@
+  toggleDrawer,
+}: DrawerPanelProps) => {
+  const { addWsEventListener } = useChrome();
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [isFilterDropdownOpen, setIsFilterDropdownOpen] = useState(false);
+  const [activeFilters, setActiveFilters] = useAtom(
+    notificationDrawerFilterAtom
+  );
+  const navigate = useNavigate();
+  const populateNotifications = useSetAtom(notificationDrawerDataAtom);
+  const addNotification = useSetAtom(addNotificationAtom);
+  const notifications = useAtomValue(notificationDrawerDataAtom);
+  const selectedNotifications = useAtomValue(notificationDrawerSelectedAtom);
+  const updateSelectedNotification = useSetAtom(updateNotificationSelectedAtom);
+  const [hasNotificationsPermissions, setHasNotificationsPermissions] =
+    useState(false);
+  const updateNotificationRead = useSetAtom(updateNotificationReadAtom);
+  const updateAllNotificationsSelected = useSetAtom(
+    updateNotificationsSelectedAtom
+  );
+  const [filterConfig, setFilterConfig] = useState<FilterConfigItem[]>([]);
+  const eventType: ChromeWsEventTypes =
+    'com.redhat.console.notifications.drawer';
+  const handleWsEvent = (event: ChromeWsPayload<NotificationsPayload>) => {
+    console.log('new event');
+    addNotification(event.data as unknown as NotificationData);
+  };
+
+  useEffect(() => {
+    let mounted = true;
+    const unregister = addWsEventListener(eventType, handleWsEvent);
+    const getNotifications = async () => {
+      try {
+        const { data } = await axios.get<{ data: NotificationData[] }>(
+          `/api/notifications/v1/notifications/drawer`,
+          {
+            params: {
+              limit: 50,
+              sort_by: 'read:asc',
+              startDate: getSevenDaysAgo(),
+            },
+          }
+        );
+        populateNotifications(data?.data || []);
+      } catch (error) {
+        console.error('Unable to get Notifications ', error);
+      }
+    };
+    const fetchPermissions = async () => {
+      const permissions = await getUserPermissions?.('notifications');
+      if (mounted) {
+        setHasNotificationsPermissions(
+          permissions?.some((item) =>
+            [
+              'notifications:*:*',
+              'notifications:notifications:read',
+              'notifications:notifications:write',
+            ].includes((typeof item === 'string' && item) || item?.permission)
+          )
+        );
+      }
+    };
+    const fetchFilterConfig = async () => {
+      try {
+        const response = await axios.get<Bundle[]>(
+          '/api/notifications/v1/notifications/facets/bundles'
+        );
+        if (mounted) {
+          setFilterConfig(
+            response.data.map((bundle: Bundle) => ({
+              title: bundle.displayName,
+              value: bundle.name,
+            }))
+          );
+        }
+      } catch (error) {
+        console.error('Failed to fetch filter configuration:', error);
+      }
```

Comment:
Same as above, we do have the JS clients available, so we can add the bundles link to it and use it here as well.